keyboards/splitkb/elora/rev1/matrix.c

```diff
@@ -0,0 +1,66 @@
+// Copyright 2024 splitkb.com (support@splitkb.com)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+#include "matrix.h"
+#include "spi_master.h"
+#include "myriad.h"
```

Comment:
This code should not be part of the matrix scan, please remove it.
```suggestion
```

Reply:
We want to hook in to the matrix if myriad is enabled. I don't really see a way how we can implement this otherwise. Or should we create a seperate myriad_matrix.c for example.

Reply:
Or another suggestion, move the `bool myriad_hook_matrix(matrix_row_t current_matrix[]);` to a seperate .h file and include that inside of the matrix.c.

Reply:
> Or another example, move the `bool myriad_hook_matrix(matrix_row_t current_matrix[]);` to a seperate .h file and include that inside of the matrix.c.

Adding a prototype in the preprocessor block below may work, too. Eg
```c
#ifdef MYRIAD_ENABLE
    bool myriad_hook_matrix(matrix_row_t current_matrix[]);
    return matrix_has_changed || myriad_hook_matrix(current_matrix);
#else
```

Reply:
Seems like a good solution and worked perfectly